pkg/builder/openapi.go

```diff
@@ -69,6 +69,21 @@ func BuildOpenAPIDefinitionsForResource(model interface{}, config *common.Config
 	return &swagger.Definitions, nil
 }
 
+// BuildOpenAPIDefinitionsForResources returns the OpenAPI spec which includes the definitions for the
+// passed type names. Currently used in the integration tests.
```

Comment:
I'd omit the last sentence, I don't consider this test code, it's generally useful.